.github/workflows/interoperability-report.yml

```diff
@@ -296,12 +296,24 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - name: "Mavennet"
-            actor: "MAVENNET_STAGING"
-          - name: "mesur.io"
-            actor: "MESUR_IO_PRODUCTION"
-          - name: "Transmute"
-            actor: "TRANSMUTE_PRODUCTION"
+          - name: "mesur.io-Transmute"
+            issuer: MESUR_IO_PRODUCTION
```

Comment:
I question why this change is happening, my understanding is that all actors are capable of all roles.

there are 3 roles, issuer, holder and verifier.

if you are presenting to a verifier, you are a holder... this change is confusing.

Reply:
We chatted, i think its best we follow up the naming conversation on an issue... in some ways, the current names are actually best, but non the less they are likely to confuse folks expecting to see "holder" when considering exchanges.

Reply:
https://github.com/w3c-ccg/traceability-interop/issues/225